UI/ClientUI.h

```diff
@@ -234,7 +226,8 @@ class ClientUI {
     std::shared_ptr<SaveFileDialog>         m_savefile_dialog;
     std::shared_ptr<PasswordEnterWnd>       m_password_enter_wnd;   //!< the authentication window
 
-    PrefixedTextures                        m_prefixed_textures;
+    std::map<std::string, std::vector<std::shared_ptr<GG::Texture>>>
```

Comment:
Would be useful to document what the key refers to. Filename? Path? Prefix? Something else?